pages/login.php

```diff
@@ -1,5 +1,3 @@
-<!-- for bootstrap css -->
-<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-eOJMYsd53ii+scO/bJGFsiCZc+5NDVN2yr8+0RDqr0Ql0h+rP48ckxlpbzKgwra6" crossorigin="anonymous">
```

Comment:
Yeah its because css should be loaded before page loads so that it can catch the css and js should be load after the dom or the page load other wise it will not be able to find that elements 

You can say that it's a convention that css should be in head tag and js should be last within body tag 

So that's why i put css in header and js in footer other wise i have to put thses two links in every file